website/docs/Queries.md

```diff
@@ -199,6 +202,53 @@ const { getByA11yValue } = render(<Component />);
 const element = getByA11yValue({ min: 40 });
 ```
 
+## TextMatch
+Some APIs accept an object containing options affecting precision of string matching as the last argument:
```

Comment:
Let's use the wording from official docs:
```suggestion

Several APIs accept a `TextMatch` which can be a `string`, `regex` or a `function` which returns `true` for a match and `false` for a mismatch.
```

Reply:
For the reset of the text I would suggest getting as close to the official docs as possible as well. See: https://testing-library.com/docs/dom-testing-library/api-queries#precision

Reply:
Do we really have 3rd options of "predicate function" at the moment? RTL does have it, but it seems that we support only string and regex args ATM.

Reply:
ATM in RNTL there are only `string` and `regex` options implemented

Reply:
```suggestion

Several APIs accept a `TextMatch` which can be a `string` or `regex`.
```